damus/Util/TimeAgo.swift

```diff
@@ -11,36 +11,45 @@ public func time_ago_since(_ date: Date) -> String {
 
     let calendar = Calendar.current
     let now = Date()
-    let unitFlags: NSCalendar.Unit = [.second, .minute, .hour, .day, .weekOfYear, .month, .year]
+    let unitFlags: NSCalendar.Unit = [.second, .minute, .hour, .day, .weekOfMonth, .month, .year]
```

Comment:
Notably `.weekOfYear` was replaced with `.weekOfMonth`. Not sure exactly the difference as far as how we're using it here, but I had to make this change because the formatter allows only the latter.